### PR TITLE
feat: Updated autoscaling api version

### PR DESCRIPTION
**Pull request by manusa**

### CODE REVIEW

File: kubernetes-model/Gopkg.lock [hunk 1]
```diff
@@ -239,12 +238,11 @@
     "k8s.io/api/authentication/v1",
     "k8s.io/api/authorization/v1",
     "k8s.io/api/autoscaling/v1",
-    "k8s.io/api/autoscaling/v2beta1",
+    "k8s.io/api/autoscaling/v2beta2",
     "k8s.io/api/batch/v1",
     "k8s.io/api/batch/v1beta1",
     "k8s.io/api/certificates/v1beta1",
     "k8s.io/api/coordination/v1",
-    "k8s.io/api/coordination/v1beta1",
```
Comment (rohanKanojia) [inline kubernetes-model/Gopkg.lock]: Why this? Was it an unused package?

Comment (manusa) [reply kubernetes-model/Gopkg.lock]: No idea (but probably), this is an autogenerated file by `dep ensure`.

**Comment by manusa** [comment]
Mind the wip label, I still have to fix a failing test.

**Comment by manusa** [comment]
> Mind the wip label, I still have to fix a failing test.

PR ready, sorry for the inconvenience

**Comment by rohanKanojia** [comment]
[merge]

**Comment by manusa** [comment]
[merge]

**Pull request merged**